charts/vela-core/templates/webhook.yaml

```diff
@@ -54,6 +98,50 @@ metadata:
   annotations:
     cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/{{ .Values.certificate.certificateName }}
 webhooks:
+  - clientConfig:
+      caBundle: Cg==
+      service:
+        name: {{ template "kubevela.name" . }}-webhook
+        namespace: {{ .Release.Namespace }}
+        path: /validating-core-oam-dev-v1alpha2-applicationconfigurations
+    failurePolicy: Fail
+    name: vappconfig.kb.io
+    rules:
+      - apiGroups:
+          - core.oam.dev
+        apiVersions:
+          - v1alpha2
+        operations:
+          - CREATE
+          - UPDATE
+        resources:
+          - applicationconfigurations
+        scope: Namespaced
+    admissionReviewVersions: 
+      - v1beta1
+    timeoutSeconds: 5
+  - clientConfig:
+      caBundle: Cg==
+      service:
+        name: {{ template "kubevela.name" . }}-webhook
+        namespace: {{ .Release.Namespace }}
+        path: /validating-core-oam-dev-v1alpha2-components
+    failurePolicy: Fail
+    name: vcomponent.kb.io
+    rules:
+      - apiGroups:
+          - core.oam.dev
+        apiVersions:
+          - v1alpha2
+        operations:
+          - CREATE
+          - UPDATE
+        resources:
+          - components
+        scope: Namespaced
```

Comment:
what is it by default?

Reply:
Default is "*". It's "Namespaced" in oam-k8s-runtime.

Reply:
OK, I see. We really need to set the scope here as Namespaced. But for workloadDefinition/traitDefinition/scopeDefinition, the scope should be `Cluster`